hack/verify-style.sh

```diff
@@ -23,8 +23,8 @@ trap 'rc=$?' ERR
 # Go to the root of the repo
 cd "$(git rev-parse --show-cdup)"
 
-GOFILES=$(find . -path ./vendor -prune -o -name '*.go' | grep -v vendor | grep -v pkg/operator/assets)
-GOPKGS=$(go list ./... | grep -v '/vendor/' | grep -v '/generated/' | grep -v pkg/operator/assets)
+GOFILES=$(find . -path ./vendor -prune -o -name '*.go' | grep -v vendor)
```

Comment:
fwiw

    GOFILES=$(find . -path ./vendor -prune -o -name '*.go' -print)

works too. (ie, explicitly telling it to print filenames that match the `-name` predicate overrides the default behavior of printing filenames that match any predicate)

Reply:
all of this is silly - filed SDN-1342 to switch to golangci-lint.